.env

```diff
@@ -1,5 +0,0 @@
-# Copyright (c) Microsoft. All rights reserved.
```

Comment:
This file is getting deleted, is that expected?

Reply:
It's not expected. Need to dig deeper into this.